verifiers/internal/gcb/keys/keys.go

```diff
@@ -0,0 +1,70 @@
+package keys
+
+import (
+	"crypto/ecdsa"
+	"crypto/x509"
+	"embed"
+	"encoding/pem"
+	"fmt"
+	"io/fs"
+	"path"
+
+	serrors "github.com/slsa-framework/slsa-verifier/errors"
+)
+
+//go:embed materials/*
+var publicKeys embed.FS
+
+type PublicKey struct {
+	Value  []byte
+	Region string
+	// TODO: key type and size
+}
+
+func PublicKeyNew(region string) (*PublicKey, error) {
+	content, err := fs.ReadFile(publicKeys, path.Join("materials", region+".key"))
+	if err != nil {
+		return nil, fmt.Errorf("%w: cannot read key materials", err)
+	}
+
+	return &PublicKey{
```

Comment:
In case someone downloads them themselves, I think the pem.Decode check for valid key should go here before sig validation

Reply:
I can move the Pem decoding here. What do you mean by "In case someone downloads them themselves"?

Reply:
+1 I think returning an error here for invalid keys would make more sense than during verification. You can still keep the bytes around if you need them.

Reply:
Done, thanks